src/redux/books/books.js

```diff
@@ -1,19 +1,18 @@
 const BOOK_ADDED = 'BOOK_ADDED';
 const BOOK_REMOVED = 'BOOK_REMOVED';
```

Comment:
- [ ] Great job defining the  `ADD_BOOK` and `REMOVE_BOOK` as action types 👏🏻 However the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. Kindly make this 👍🏻